trusted-firmware-m/platform/CMakeLists.txt

```diff
@@ -149,6 +149,7 @@ target_compile_definitions(platform_region_defs
     INTERFACE
         $<$<BOOL:${BL2}>:BL2>
         $<$<BOOL:${SECURE_UART1}>:SECURE_UART1>
+        $<$<BOOL:${NRF_NS_STORAGE}>:NRF_NS_STORAGE>
```

Comment:
Ideally this should be moved to nrf-specific CMakeLists.txt file, otherwise, this needs to be upstream-ed.

Reply:
This is because this is not a generic platform-level option, but rather a Nordic-specific.